DBAL/Types/UtcDateTimeType.php

```diff
@@ -34,7 +34,7 @@ public function convertToDatabaseValue($date, AbstractPlatform $platform): ?stri
             return null;
         }
 
-        if ($date instanceof DateTime) {
+        if ($date instanceof \DateTimeInterface) {
```

Comment:
Hello! Thx for this fix
Something has been changed from past version :)
https://github.com/doctrine/dbal/blob/v2.6.0/lib/Doctrine/DBAL/Types/DateTimeType.php#L56